neuro_evolution_ctrnn/brain_visualizer/__init__.py

```diff
@@ -1,40 +1,260 @@
-import tkinter as tk
+from brain_visualizer.position import Positions
+from brain_visualizer.weights import Weights
+from brain_visualizer.neurons import Neurons
+from brain_visualizer.events import Events
+
+import json
+
+import pygame, sys
+from pygame.locals import *
 
 
 class BrainVisualizerHandler(object):
     def __init__(self):
         self.current_visualizer = None
 
     def launch_new_visualization(self, brain):
-        self.current_visualizer = BrainVisualizer(brain)
+        self.current_visualizer = PygameBrainVisualizer(brain)
         return self.current_visualizer
 
 
-class BrainVisualizer(object):
-
+class PygameBrainVisualizer(object):
     def __init__(self, brain):
         self.brain = brain
 
-        root = tk.Tk()
-        self.root = root
-        self.app = Window(root)
-        root.wm_title("Tkinter window")
-        root.geometry("200x200")
+        # Read Configuration file
+        with open("../CTRNN_Simulation_Results/data/" + ((sys.argv[2])[-19:]) + "/Configuration.json", "r", encoding='utf-8') as read_file:
```

Comment:
Das muss anders gemacht werden, es sollte nicht davon ausgegangen werden dass das 3. Element in den Argumenten der Trainingsordner ist. Am besten über ein Konstruktorargument `dir` den Ordner verlangen und hier benutzen. Ist mir gerade beim Testen aufgefallen.

btw was macht die -19 hier?